install/kubernetes/helm/istio/templates/crds.yaml

```diff
@@ -143,10 +143,10 @@ spec:
   scope: Cluster
   version: v1alpha1
 ---
-# {{- end }}
+{{- end }}
 
 # these CRDs only make sense when security is enabled
-# {{- if or .Values.security.enabled .Values.global.useMCP }}
+{{- if or .Values.security.enabled .Values.global.useMCP }}
```

Comment:
I think we can remove all of these conditionals as they make upgrading harder to deal with.